tests/rules/metavar_ellipsis_param.yaml

```diff
@@ -0,0 +1,13 @@
+rules:
+  - id: test
+    languages:
+      - go
+    message: javascript inside html working!
```

Comment:
Maybe choose something more generic or something that better describes this case?